app/script/localization/Localizer.js

```diff
@@ -70,6 +78,53 @@ class Localizer {
 
 z.localization.Localizer = new Localizer();
 
+z.l10n = (() => {
+
+  function replaceWithString(string, substitute) {
+    return string.replace(/{{\w+}}/, substitute);
+  }
+
+  function replaceWithObject(string, substitute) {
+    for (let identifier in substitute) {
+      if (substitute.hasOwnProperty(identifier)) {
+        string = string.replace(`{{${identifier}}}`, substitute[identifier]);
```

Comment:
Sure i could do that was just thinking if we have situations like this. But sure i can do it :)